src/EnergyPlus/StandardRatings.cc

```diff
@@ -1481,23 +1481,23 @@ namespace StandardRatings {
     void CalcTwoSpeedDXCoilRating(EnergyPlusData &state,
                                   std::string const &DXCoilName,
                                   std::string const &DXCoilType,
-                                  int const &DXCoilType_Num,
+                                  int const DXCoilType_Num,
```

Comment:
Here are the examples of const reference parameters that should be converted to value parameters. I think I made this comment on the PR, but it was ignored.